contrib/templates/default-scala/template/{{.project_name}}/src/main/scala/com/examples/Main.scala

```diff
@@ -12,6 +12,11 @@ object Main {
   def main(args: Array[String]): Unit = {
     println("Hello, World!")
 
+    val catalog = getFromArgs(args, "catalog").getOrElse("samples")
+    val schema = getFromArgs(args, "schema").getOrElse("nyctaxi")
+
+    println(s"Using catalog: $catalog, schema: $schema")
```

Comment:
This still requires some code to actually set the default catalog/schema to some other value. I think that might also just be `spark.sql("USE catalog")` as seen in the Python verison?